lib/good-influx-http.js

```diff
@@ -28,11 +67,14 @@ class GoodInfluxHttp extends GoodHttp {
             payload: lineData.join('\n')
         };
 
-        Hoek.merge(wreckOptions, this._settings.wreck, false);
+        Hoek.merge(wreckOptions, this._settings.wreck, { nullOverride: false });
 
         // Prevent this from user tampering
-        wreckOptions.headers['content-type'] = 'text/plain';
-        Wreck.request('post', this._endpoint, wreckOptions, callback);
+        wreckOptions.headers ? wreckOptions.headers['content-type'] = 'text/plain'
+            : wreckOptions.headers = { 'content-type': 'text/plain' };
+        Wreck.request('post', this._endpoint, wreckOptions).then((req) => {
```

Comment:
New Wreck doesn't allow for callback, so instead, I duplicated the old wreck behavior.